sem-reference_5b6968642c7d3a03f89d6bff.md

```diff
@@ -1,2 +1,350 @@
 # sem commad line tool Reference
 
+* [Overview](#overview-of-sem)
+* [Syntax](#syntax)
+* [Operations](#operations)
+* [Resource types](#resource-types)
+* [Working with organizations](#working-with-organizations)
+* [Working with resources](#working-with-resources)
+* [Project initialization](#project-initialization)
+* [Help commands](#help-commands)
+* [Flags](#flags)
+* [Examples](#examples)
+* [See also](#see-also)
+
+## Overview of sem
+
+`sem` is the command line interface to Semaphore 2.0. This reference page
+covers the syntax and the commands of `sem` and presents examples of the `sem`
+commands.
+
+## Syntax
+
+The general syntax of the `sem` utility is as follows:
+
+    sem [COMMAND] [RESOURCE] [NAME] [flags]
+
+where `[COMMAND]` is the name of the command as explained in this reference
+page, `[RESOURCE]` is the resource type that interests us, `[NAME]` is the
+actual name of the resource and `[flags]` are optional flags.
+
+Not all `sem` commands require a `[RESOURCE]` value and most `sem` commands do
+not have a `[flag]` part.
+
+Some of the `[flags]` values need an additional argument, as it is the case
+with the `-f` flag, which requires a valid path to a local file.
+
+## Operations
+
+The following list briefly describes the `sem` operations:
+
+* *connect*: The `connect` command is used for connecting to organizations for the first time.
+* *context*: The `context` command is used for adding an organization to your existing ones for the first time.
+* *create*: The `create` command is used for creating new resources.
+* *delete*: The `delete` command is used for deleting existing resources.
+* *describe*: The `describe` command is used for getting information about existing resources.
+* *get*: The `get` command is used for getting the list of an existing type of resource.
+* *help*: The `help` command is used for getting help about `sem` or an existing `sem` command.
+* *init*: The `init` command is used for adding an existing GitHub repository to Semaphore 2.0 for the first time.
+* *version*: The `version` command is used for getting the version of the `sem` utility.
+
+## Resource types
+
+Semaphore 2.0 supports two types of resources: `secrets` buckets and Semaphore
+2.0 projects. Most resource related operations require a resource name.
+
+## Working with organizations
+
+This group of `sem` commands for working with organizations contains the
+`sem connect` and `sem context` commands.
+
+### sem connect
+
+The `sem connect` command allows you to connect to a Semaphore 2.0 organization
+for the first time and requires two command line arguments. The first command
+line argument is the organization name and the second command line argument is
```

Comment:
> organization name

Since relatively recently this is called "Organization ID"

<img width="682" alt="screen shot 2018-08-21 at 7 26 38 pm" src="https://user-images.githubusercontent.com/20469/44418270-3ba30c80-a578-11e8-9e2f-1555de14edb8.png">

Reply:
Changed it to `domain`.